hist/histv7/inc/ROOT/RHistUtils.hxx

```diff
@@ -42,6 +42,10 @@ struct RCoordArray: std::array<double, DIMENSIONS> {
 
    /// Construction from a C-style array.
    RCoordArray(double (&arr)[DIMENSIONS]): Base_t(arr) {}
+
+   /// Copy-construction from a C++-style array.
+   /// (No need for a move-constructor, it isn't any better for doubles)
+   RCoordArray(const std::array<double, DIMENSIONS>& arr): Base_t(arr) {}
```

Comment:
...and C++20 `span<double, DIMENSIONS>` will fix that even better. But for now this is perfect!